modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -59,4 +59,58 @@ test.describe('Manage object fields through Model Builder', () => {
 			ListTypeDefinition.id
 		);
 	});
+
+	test('can delete object field', async ({
+		apiHelpers,
+		modelBuilderPage,
+		page,
+		viewObjectDefinitionsPage,
+	}) => {
+		await page.goto('/');
+
+		const objectDefinition =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition('default');
+
+		await viewObjectDefinitionsPage.goto();
+
+		await viewObjectDefinitionsPage.openObjectFolder('default');
+
+		await viewObjectDefinitionsPage.viewInModelBuilder();
+
+		await modelBuilderPage.leftSidebarItems
+			.filter({hasText: objectDefinition.name})
+			.click();
+
+		await modelBuilderPage.createObjectField({
+			mandatory: false,
+			objectDefinitionName: objectDefinition.name,
+			objectFieldBusinessType: 'Integer',
+			objectFieldLabel: 'intField',
+		});
```

Comment:
Hey @rafaellajordao, I don't see need to create a object field by UI. Since the main goal of the test is the delete action, we can create this field through API. You can check the ApiHelpers file and see if you function was already created.

Reply:
Hey @igor-franca,
The only change I couldn't make was changing the creation of the field from UI to API, I looked in the files but couldn't find it. Can you show me how I can do this? :pleading_face:

Reply:
> Hey @igor-franca, The only change I couldn't make was changing the creation of the field from UI to API, I looked in the files but couldn't find it. Can you show me how I can do this? 🥺

Hey, @rafaellajordao I can show you how to do it, I sent you a message on slack to talk about this point 😄